### DISCUSSION TITLE
Can this package work with `visual-fill-column-center-text' set to t?

### DISCUSSION THREAD

**Discussion by zyd-n** [minad/org-modern#202] [Q&A] [answered]
I use `visual-fill-column-mode` with `visual-fill-column-center-text` set to `t`. This works well to center the text of a buffer, which is what I do when viewing a single org buffer on a large screen. However, it doesn't play nice with org-modern: quote, source, verse and other blocks that use the line drawing box are broken, appearing on the far left of the screen unless I set the center-text option to nil. See attached screenshots at end for demonstration.

Can this package work with this setting or perhaps there's another "center the text in a buffer" package that is likely to be compatible?

***

![org-modern1](https://github.com/minad/org-modern/assets/169315939/9300cb01-f8c1-45ec-a0db-2f1be679d87f)
![org-modern2](https://github.com/minad/org-modern/assets/169315939/7757b8ee-abf7-4f11-b9c5-f0ac27b3f2d1)

**Comment by minad** [comment, marked as answer]
You can disable the line in the fringe by setting org-modern-block-fringe=nil. Alternatively try https://github.com/jdtsmith/org-modern-indent.

**Reply by zyd-n** [reply]
Ah, didn't realize it was drawn in the fringe. I don't use indentation in Org, so I don't think that package will work. Though I like the block drawings, disabling them will work. Thanks! And apologies for opening this first as an issue, didn't realize the discussions was actively used.
